src/selectors/index.js

```diff
@@ -15,15 +15,16 @@ import {
   combNetworkMSBNDescription,
 } from './combiners';
 
-export const getNetwork = state => state.network;
+export const getNetwork = prop('network');
 export const getNodes = state => state.network.nodes || state.nodes || [];
```

Comment:
You can you [either](https://ramdajs.com/docs/#either) here:
```js
export const getNodes = either(path(['network', 'nodes']), propOr([], 'nodes'));
```

Reply:
Great, thx